lib/node_modules/@stdlib/math/base/special/cpolar/test/test.assign.js

```diff
@@ -0,0 +1,107 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var cabs = require( '@stdlib/math/base/special/cabs' );
+var Complex128 = require( '@stdlib/complex/float64' );
+var Float64Array = require( '@stdlib/array/float64' );
+var cphase = require( '@stdlib/math/base/special/cphase' );
+var cpolar = require( './../lib/assign.js' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof cpolar, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function computes the absolute value and phase of a complex number', function test( t ) {
+	var expected;
+	var out;
+	var v;
+
+	out = [ 0.0, 0.0 ];
+
+	v = cpolar( new Complex128( 5.0, 3.0 ), out, 1, 0 );
+	expected = [ cabs( new Complex128( 5.0, 3.0 ) ),
+		cphase( new Complex128( 5.0, 3.0 ) ) ];
```

Comment:
```suggestion
	expected = [
		cabs( new Complex128( 5.0, 3.0 ) ),
		cphase( new Complex128( 5.0, 3.0 ) )
	];
```

Fine to include linebreaks, but they should be on separate lines from the brackets.